### PR TITLE
Fixed failed test due to keepRecordsInState set to true by JRS

### PR DESCRIPTION
**Pull request by JeffreyDallas**
**Related issue(s)**:
#00934

**Summary of the change**:
The network error regression test because test
accountsGetPayerRecordsIfSoConfigured only works when 
keepRecordsInState is set to false before it starts.

JRS changed keepRecordsInState to true before launch the client, so
we need refactor this test case

### CODE REVIEW

File: test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java [hunk 1]
```diff
@@ -58,7 +58,10 @@ public static void main(String... args) {
 	private HapiApiSpec accountsGetPayerRecordsIfSoConfigured() {
 		return defaultHapiSpec("AccountsGetPayerRecordsIfSoConfigured")
 				.given(
-						cryptoCreate("payer")
+						cryptoCreate("payer"),
```
Comment (Neeharika-Sompalli) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java:61]: LGTM. But, @anighanta had a PR open with changes to this Suite. Not sure if that is merged

**Review by Neeharika-Sompalli — APPROVED** [review]
LGTM

File: test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java [hunk 2]
```diff
@@ -58,7 +58,10 @@ public static void main(String... args) {
...
+						fileUpdate(APP_PROPERTIES)
+								.payingWith(ADDRESS_BOOK_CONTROL)
+								.overridingProps(Map.of("ledger.keepRecordsInState", "false"))
```
Comment (anighanta) [inline test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java:64]: This step in done in then().

Comment (JeffreyDallas) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java:64]: I will check the other related PR to make change if necessary

Comment (JeffreyDallas) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java:64]: In JRS, "ledger.keepRecordsInState" was set to true before launch the client.
This test case accountsGetPayerRecordsIfSoConfigured  can only pass if before 
ledger.keepRecordsInState is set to false.

Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java:64]: sorry for the late reply Jeffrey, as by default now that ledger.keepRecordsInState=true, when this is set to false accounts in records must be zero. That is why I set this value false in when(). You are just repeating the transaction in given. So it just will be repeated operation in when()

Comment (JeffreyDallas) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java:64]: Got it, I will revert the change later

Comment (anighanta) [reply test-clients/src/main/java/com/hedera/services/bdd/suites/records/RecordCreationSuite.java:64]: But, was it still failing with my changes in ?

**Pull request merged**